src/blocks/app-and-main-menu/_fhi-header.block.scss

```diff
@@ -0,0 +1,128 @@
+.fhi-header {
+    margin-bottom: $fhi-space-6;
+
+    &__brand {
+        padding: $fhi-brand-padding-top-xs 0 $fhi-brand-padding-bottom-xs;
+    }
+    &__brand-container {
+        display: block;
+        color: $body-color;
+        font-size: $fhi-font-size-2;
+        position: relative;
+        height: $fhi-brand-container-height;
+        text-decoration: none;
+
+        &:hover {
+            color: $body-color;
+            text-decoration: none;
+        }
+    }
+    &__brand-content {
+        margin-left: $fhi-container-padding-xs;
+        max-width: calc(100% - #{$fhi-px * 100});
+        z-index: 1;
+    }
+    &__logo {
+        display: inline-block;
+        width: $fhi-px * 82;
+        height: $fhi-brand-container-height;
+    }
+    &__logo-gfx {
```

Comment:
`icon` instead of `gfx` maybe?

Reply:
Well, is this really an icon?

Reply:
On second thought, since it is generated in the icon generating system I agree.

Reply:
On second thought, since it is generated in the icon generating system I agree.